src/components/RoomView/gameLog.tsx

```diff
@@ -1,21 +1,29 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { useSelector } from 'react-redux';
 import './roomView.css';
 import { RootState } from '../../store';
 
 const GameLog: React.FC = () => {
+  const divRef = useRef<null | HTMLDivElement>(null);
   const gameHistory = useSelector((state: RootState) => state.game.gameHistory);
   const team1 = useSelector((state: RootState) => state.teamOne);
   const team2 = useSelector((state: RootState) => state.teamTwo);
-  //here we are comparing if the teamSubmitted(which is the teamId, a number) in singleHistory===teamOne.team1Id
-  //previously we have playerSubmitted in Cluetype,
-  //but if we compare playerSubmitted(which is the spymaster's playerId) and teamOne.teamOneSpymaster.playerId
-  //when the spymaster left the room, the playerId will be ''
-  //and the game log will disappear because the logic will be false
+  // here we are comparing if the teamSubmitted(which is the teamId, a number) in singleHistory===teamOne.team1Id
+  // previously we have playerSubmitted in Cluetype,
+  // but if we compare playerSubmitted(which is the spymaster's playerId) and teamOne.teamOneSpymaster.playerId
+  // when the spymaster left the room, the playerId will be ''
+  // and the game log will disappear because the logic will be false
```

Comment:
this is just TS lint make me give comments a space between slashes and sentences.